notebooks/04_model_select_and_optimize/hypertune_spark_deep_dive.ipynb

```diff
@@ -76,7 +77,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 60,
+   "execution_count": 37,
    "metadata": {},
    "outputs": [
     {
```

Comment:
About this:
```
%env PYSPARK_PYTHON=/anaconda/envs/recommender/bin/python
%env PYSPARK_DRIVER_PYTHON=/anaconda/envs/recommender/bin/python
```
In the other spark notebooks we are not adding this since it is explained in the setup. In case you still think it is important to have it, maybe we should add a comment saying that you have to add the executor of your environment.

Reply:
Removed. Good catch